### ISSUE TITLE
[suggestion] thaumcraft shield variants

### ISSUE THREAD

**Issue by varvolgyibeni** [Azanor/thaumcraft-suggestions#203]
normal shields break too quickly,  having thaumium, void metal or thaumium fortress shields would be useful, also an elemental shield, which either knocks back enemies a lot, or has a thorns effect.

**Comment by AnrDaemon** [comment]
Shield**S** ? You sure there's multiple shields?

**Comment by varvolgyibeni** [comment]
yes. you can craft more than one shield. also, it's "there are multiple" not "there is multiple"

**Comment by AnrDaemon** [comment]
I only see one shield.
One.
Shield.
No variants whatsoever.

**Comment by varvolgyibeni** [comment]
Sure there are no variants, but as i already said, you can have more than one, if i said "the normal shield breaks too quickly" it would've sounded like i only talked about one specific shield and not all shields in general.

**Comment by AnrDaemon** [comment]
Again, I only see ONE shield. THE_SHIELD. Did something change and MC now allows for different shields to exist?

**Comment by varvolgyibeni** [comment]
Then you really are an imbecile. Please stop writing here, this github isn't a place for you to prove your faulty knowledge of the english language. Let me explain this for you: when you have 3 shields in your inventory, then it's plural because there are multiple of them! The english language is wonderful, isn't it?

**Comment by AnrDaemon** [comment]
Calling names is the best argument you can find?
No screenshots of "3 shields in inventory" ?

**Comment by varvolgyibeni** [comment]
https://imgur.com/a/zUeFN Look! 3 shields! Not 3 of "THE_SHIELD" please learn grammar before you start a pointless argument.

**Comment by AnrDaemon** [comment]
That's the same shield type. Go learn to minecraft already.

**Comment by varvolgyibeni** [comment]
Same shield type, yet there are multiple of them, therefore you say "shields".Do you say the same for diamond swords as well? Like "I crafted 2 of "THE_DIAMOND_SWORD"? Can we please stop arguing now? This is supposed to be the github page of thaumcraft, not the place for 2 numbnuts to dispute the correct grammar for minecraft shields.

**Comment by AnrDaemon** [comment]
You can implement an item that looks like shield and behave like shield, but minecraft will not consider it shield still, and not apply any shield mechanics to it by itself.
If you are unable to grasp the concept of programming restriction that simply not allow for existence of different shield types in current minecraft, this is going nowhere.

**Comment by varvolgyibeni** [comment]
Can you please further elaborate on what you said in the first sentence? What do you mean "not apply any shield mechanics by itself"? And if an item looks and acts like a shield, then fundamentally it's a shield. Also, i did not know there were "programming restrictions", you could also explain those, that would be swell. I thought there already are mods that have different shields, like Silent's gems mod or thermal expansion or thermal foundation (i don't know which), or the spartan shields, Immersive engineering also adds a shield, and a mod that adds police shields, but i cannot recall it's name.

**Comment by AnrDaemon** [comment]
Fundamentally it will NOT be a shield. Let me make an example.
There's a "sword type", you can create Thaumium sword, tell Minecraft "Hey, dude! It's a kind of sword! Just bluish and darkish! But it's a sword all right! No kidding!", Minecraft will happily swallow it and apply all sword mechanics to it, like parrying and charged attacks and what not.
But there's no "shield type". There's only The Shield. Single specific item. All checks for shield-related mechanics in Minecraft done against this, exact, SPECIFIC ITEM ID, not any item with some specific attribute(type).
Yes, Mojang is retarded like that.

**Comment by varvolgyibeni** [comment]
Ah, i see, thanks for explaining it. Good day to you, sir.
